utility_functions/main.cpp

```diff
@@ -2,4 +2,9 @@
 
 int main() {
     // make sure to run your functions!
+    Sum(7,10);
 }
+
+std::vector<int> v{1,2,3,4};
+
+std::vector<int> VectorPlusN(v, 5);
```

Comment:
This should be included in main so the test runs. Also, std::vector<int> VectorPlusN(v, 5); is invalid syntax.
VectorPlusN() has a return that has not been instantiated. Use td::vector<int> returned_vector = VectorPlusN(v, 5);